src/Voron/Data/BTrees/Tree.Stream.cs

```diff
@@ -251,7 +259,10 @@ public VoronStream ReadStream(Slice key)
             using (var it = tree.Iterate())
             {
                 if (it.Seek(0) == false)
-                    return null;
+                {
+                    Debug.Assert(false);
```

Comment:
It is hard to trace failing debug asserts when there is no message.

Reply:
We are actually checking a bit earlier that this can't happen.